synapse/crypto/keyring.py

```diff
@@ -638,7 +638,6 @@ def get_server_verify_key_v2_indirect(
                         for server_name, server_keys in keys_to_fetch.items()
                     }
                 },
-                long_retries=True,
```

Comment:
I'd prefer to explicitly set `long_retries=False` rather than relying on the default tbh.

Reply:
as discussed in #synapse-dev
* we don't set `long_retries=False` anywhere else
* given how long `long_retries` takes, you don't want to set it unless you *really* want it. Accordingly I think it makes sense for the default to be `False` and for everything to rely on that being the default.